rolepermissions/roles.py

```diff
@@ -34,15 +34,26 @@ def get_roles(cls):
 
 class RolesClassRegister(type):
 
+    @staticmethod
+    def is_abstract(meta):
+        if meta is not None:
+            return getattr(meta, 'abstract', False)
+        else:
+            return False
+
     def __new__(cls, name, parents, dct):
+        meta = dct.pop("Meta", None)
         role_class = super(RolesClassRegister, cls).__new__(cls, name, parents, dct)
```

Comment:
Hey, @fjsj using `pop` in here makes the super being called without referencing the Meta class.

Also, it makes, the reference to `.Meta` vanishes:
![image](https://github.com/vintasoftware/django-role-permissions/assets/8262459/88169c24-d860-4617-854b-4eb1e350b9ff)

I suggest changing it to `.get`

Reply:
Not sure If there is any context for using `pop` instead of  `get`, there is anything @wieczorek1990?

Reply:
Good catch. I would say keep the old code if possible.

Reply:
@dennys-bd I wrote it the way Django does. It works like this: you mark one class as abstract and the additional Meta properties are not shared.

Reply:
@wieczorek1990 I see, although I'm not 100% aware of how it works in Django, in the way that it was written It will never share Meta properties, independently if it is an abstract class or not

Are you referring to this?
https://github.com/django/django/blob/ee36e101e8f8c0acde4bb148b738ab7034e902a0/django/db/models/base.py#L110

If so meta will be re-added in the newly created class, in the lines below